presto-common/src/main/java/com/facebook/presto/common/RuntimeStats.java

```diff
@@ -93,9 +102,8 @@ public void mergeWith(RuntimeStats stats)
         if (stats == null) {
             return;
         }
-        stats.getMetrics().values().forEach(metric -> {
-            metrics.computeIfAbsent(metric.getName(), RuntimeMetric::new);
-            metrics.get(metric.getName()).mergeWith(metric);
+        stats.getMetrics().forEach((name, newMetric) -> {
+            metrics.computeIfAbsent(name, RuntimeMetric::new).mergeWith(newMetric);
```

Comment:
same nit; we don't need `{..}` if it's single statement

Reply:
Done.